backend/tests/clubs/test_ticketing.py

```diff
@@ -947,6 +947,72 @@ def test_get_cart_replacement_required_sold_out(self):
         to_add = set(map(lambda t: str(t.id), tickets_to_add))
         self.assertEqual(len(in_cart & to_add), 0, in_cart | to_add)
 
+    def test_place_hold_on_tickets(self):
+        from clubs.views import TicketViewSet
+
+        self.client.login(username=self.user1.username, password="test")
+
+        # Add a few tickets
+        cart, _ = Cart.objects.get_or_create(owner=self.user1)
+        tickets_to_add = self.tickets1[:3]
+        for ticket in tickets_to_add:
+            cart.tickets.add(ticket)
+        cart.save()
+
+        TicketViewSet._place_hold_on_tickets(self.user1, cart.tickets)
+        holding_expiration = timezone.now() + timezone.timedelta(minutes=10)
+
+        for ticket in cart.tickets.all():
+            self.assertIsNone(ticket.owner)
+            self.assertEqual(self.user1, ticket.holder)
+            self.assertAlmostEqual(
+                holding_expiration,
+                ticket.holding_expiration,
+                delta=timedelta(seconds=10),
+            )
+
+        # Move Django's internal clock 10 minutes forward
+        with freezegun.freeze_time(holding_expiration):
+            Ticket.objects.update_holds()
+            for ticket in cart.tickets.all():
+                self.assertIsNone(ticket.owner)
+                self.assertIsNone(ticket.holder)
+                # TODO: Ask whether to reset holding expiration to None
```

Comment:
Can remove this.

Reply:
Separately, this decision's up to you. We didn't null it because there won't be a case where we rely on `holding_expiration` without checking `holder!=None`, but it is a bit unsafe.

Reply:
Was the `TODO` comment removed?